pkg/pulsar/utils/sink_config.go

```diff
@@ -44,6 +44,8 @@ type SinkConfig struct {
 	InputSpecs            map[string]ConsumerConfig `json:"inputSpecs,omitempty" yaml:"inputSpecs"`
 	Configs               map[string]interface{}    `json:"configs,omitempty" yaml:"configs"`
 
+	CustomRuntimeOptions string `json:"customRuntimeOptions,omitempty" yaml:"customRuntimeOptions"`
```

Comment:
We should keep the same name with https://github.com/streamnative/pulsarctl/blob/master/pkg/pulsar/utils/source_config.go#L44.